scripts/large_models/chainer_chain.py

```diff
@@ -10,7 +10,7 @@
 from chainer.links.connection.convolution_2d import Convolution2D
 from chainer.links.connection.linear import Linear
 from chainer.functions.noise.dropout import dropout
-from chainer.functions.pooling.max_pooling_2d import max_pooling_2d
+from chainer.functions.pooling.max_pooling_nd import max_pooling_2d
```

Comment:
This is fine, but cannot we use `chainer.functions import max_pooling_2d` so we can support both old chainer and new one?

Reply:
Thank you, fixed.